src/SFA.DAS.PR.Jobs/MessageHandlers/EmployerAccounts/CreatedAccountEventHandler.cs

```diff
@@ -43,4 +43,4 @@ public async Task Handle(CreatedAccountEvent message, IMessageHandlerContext con
         }
         await _providerRelationshipsDataContext.SaveChangesAsync(context.CancellationToken);
     }
-}
+}
```

Comment:
I like to see empty lines at the end of file, git hub likes to have them as well.